vms/platformvm/genesis/genesis.go

```diff
@@ -22,6 +24,8 @@ type Genesis struct {
 	Timestamp     uint64    `serialize:"true"`
 	InitialSupply uint64    `serialize:"true"`
 	Message       string    `serialize:"true"`
+
+	GenesisID ids.ID `serialize:"false"`
```

Comment:
we're changing a genesis related object but as long as changes are not serialized it's fine

Reply:
Kinda want to see if we can remove this field, wrote a comment in `state` related to this